corehq/apps/userreports/templates/userreports/edit_data_source.html

```diff
@@ -68,6 +68,11 @@ <h5 class="modal-title">{% trans 'Are you sure you want to delete this data sour
       {% endif %}
       <a href="{% url 'preview_configurable_data_source' domain data_source.get_id %}" class="btn btn-default">{% trans 'Preview data' %}</a>
       <a href="{% url 'summary_configurable_data_source' domain data_source.get_id %}" class="btn btn-default">{% trans 'Data Source Summary' %}</a>
+      <a href="{% url 'configurable_data_source_json' domain data_source.get_id %}"
+         class="btn btn-default track-usage-link"
```

Comment:
just confirming that `track-usage-link` class and other data attrs make the tracking happen?

Reply:
yep: https://github.com/dimagi/commcare-hq/blob/99187d23fbf7669c3e816a223fa92273bab0c830/corehq/apps/hqwebapp/static/hqwebapp/js/main.js#L350-L354

Reply:
yep: https://github.com/dimagi/commcare-hq/blob/99187d23fbf7669c3e816a223fa92273bab0c830/corehq/apps/hqwebapp/static/hqwebapp/js/main.js#L350-L354